types/index.d.ts

```diff
@@ -1,21 +1,84 @@
+/**
+ * A database transaction
+ */
```

Comment:
Transaction, Schema and config could all probably benefit from @namespace and @property https://jsdoc.app/tags-property.html